gmso/formats/lammpsdata.py

```diff
@@ -183,48 +183,159 @@ def write_lammpsdata(topology, filename, atom_style="full"):
             # Pair coefficients
             data.write("\nPair Coeffs # lj\n\n")
             for idx, param in enumerate(topology.atom_types):
-                data.write(
-                    "{}\t{:.5f}\t{:.5f}\n".format(
-                        idx + 1,
-                        param.parameters["epsilon"]
-                        .in_units(u.Unit("kcal/mol"))
-                        .value,
-                        param.parameters["sigma"].in_units(u.angstrom).value,
-                    )
-                )
+                # expected expression for lammps for standard LJ
+                lj_expression = "4.0 * epsilon * ((sigma/r)**12 - (sigma/r)**6)"
 
-            if topology.bonds:
-                data.write("\nBond Coeffs\n\n")
-                for idx, bond_type in enumerate(topology.bond_types):
+                if sympify(param.expression).equals(sympify(lj_expression)):
                     data.write(
                         "{}\t{:.5f}\t{:.5f}\n".format(
                             idx + 1,
-                            bond_type.parameters["k"]
-                            .in_units(u.Unit("kcal/mol/angstrom**2"))
-                            .value
-                            / 2,
-                            bond_type.parameters["r_eq"]
-                            .in_units(u.Unit("angstrom"))
+                            param.parameters["epsilon"]
+                            .in_units(u.Unit("kcal/mol"))
+                            .value,
+                            param.parameters["sigma"]
+                            .in_units(u.angstrom)
                             .value,
                         )
                     )
-
-            if topology.angles:
-                data.write("\nAngle Coeffs\n\n")
-                for idx, angle_type in enumerate(topology.angle_types):
+                elif (
+                    scaling_factor := simplify(lj_expression)
```

Comment:
also, the walrus operator (`:=`) is a new thing in python >= 3.8, but since we are still trying to support python 3.7, this may be in appropriate. 
```suggestion
                    scaling_factor = simplify(lj_expression)
```